sound/smooth_swing_v2.h

```diff
@@ -8,36 +8,53 @@
 class SmoothSwingV2 : public SaberBasePassThrough {
 public:
   SmoothSwingV2() : SaberBasePassThrough() {}
-
+  
   void Activate(SaberBase* base_font) {
     STDOUT.println("Activating SmoothSwing V2");
     SetDelegate(base_font);
     if (swingl.files_found() != swingh.files_found()) {
       STDOUT.println("Warning, swingl and swingh should have the same number of files.");
     }
     swings_ = std::min<size_t>(swingl.files_found(), swingh.files_found());
+    if (swng.files_found() > 0)
+    {
```

Comment:
Please put the { at the end of the previous line, like the rest of this code does.